src/server_config.rs

```diff
@@ -1,9 +1,11 @@
 use super::*;
 
 #[derive(Clone, Default)]
```

Comment:
I checked and we can remove this `Clone`:

```suggestion
#[derive(Default)]
```